### DISCUSSION TITLE
[Feature Request] Multiple Notepads

### DISCUSSION THREAD

**Discussion by pinetreemoonlight** [bluecaret/carettab#338] [Ideas and Feature Requests]
It would be great if we could add multiple (separate) notepads to the page.

**Comment by bluecaret** [comment]
I'm happy to inform you that this will be a feature in the next update! In fact, with Premium Access you'll be able to add an infinite* number of widgets. Currently looking at getting the next update out in the next few weeks. Cheers!

* as storage space allows
